src/migrations/20240126095544-add-flag-trends.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+exports.up = function(db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS flag_trends (
+            id VARCHAR(255) NOT NULL,
+            project VARCHAR(255) NOT NULL REFERENCES projects (id) ON DELETE CASCADE,
+            total_flags INTEGER NOT NULL,
+            stale_flags INTEGER NOT NULL,
```

Comment:
we separate stale flags marked by user as stale vs potentially stale flags by their age. We can calculate active flags by subtracting stale and potentially stale from total